load-test/scenarios/data/generate-pins.csv

```diff
@@ -0,0 +1,2 @@
+pupil[0],pupil[1],pupil[2],pupil[3],pupil[4],pupil[5],pupil[6],pupil[7],pupil[8],pupil[9],pupil[10],pupil[11],pupil[12],pupil[13],pupil[14],pupil[15],pupil[16],pupil[17],pupil[18],pupil[19],pupil[20],pupil[21],pupil[22],pupil[23],pupil[24],pupil[25],pupil[26],pupil[27],pupil[28],pupil[29],pupil[30],pupil[31],pupil[32],pupil[33],pupil[34],pupil[35],pupil[36],pupil[37],pupil[38],pupil[39],pupil[40],pupil[41],pupil[42],pupil[43],pupil[44],pupil[45],pupil[46],pupil[47],pupil[48],pupil[49],pupil[50],pupil[51],pupil[52],pupil[53],pupil[54],pupil[55],pupil[56],pupil[57],pupil[58],pupil[59],pupil[60],
```

Comment:
are you going to generate this on the fly as we discussed?

Reply:
@GuyHarwood for scenarios around pupil load testing 100% yes.
I believe this file is meant to be used for admin perf test around generating pins. 
@sinhaamit1980 do you need this file?
If not, are you testing generate pins post requests from the admin app?